leakcanary-android-core/src/main/java/leakcanary/DefaultOnHeapAnalyzedListener.kt

```diff
@@ -55,6 +73,55 @@ class DefaultOnHeapAnalyzedListener(private val application: Application) : OnHe
     )
   }
 
+  /**
+   * Android TV devices do not have notifications, therefore the only easy and non-invasive way
+   * to communicate with user is via Toast messages. These are used just to grab user attention and
+   * to direct them to Logcat where a much more detailed report will be printed.
+   */
+  private fun showToast(heapAnalysis: HeapAnalysis) {
+    handler.post {
+      val resumedActivity = InternalLeakCanary.resumedActivity
+      if (resumedActivity == null) {
+        SharkLog.d { "Can't display Toast, activity is backgrounded" }
```

Comment:
Devs have no idea what toast we're trying to display. Not sure it's worth logging, but if we do we should be more specific.